configs/bokeh.json

```
@@ -1,35 +1,88 @@
 {
   "index_name": "bokeh",
   "start_urls": [
-    "https://docs.bokeh.org"
+    {
+      "url": "https://docs.bokeh.org/(?P<lang>.*?)/(?P<version>.*?)/user_guide/",
+      "selectors_key": "user_guide",
+      "page_range": 3,
```

```suggestion
      "page_rank": 3,
```
Did you mean this parameter?